[section#terminate
    [:en] [`terminate]
]

[%@library terminate]
[codeblock:declaration]
[[noreturn]] void terminate() noexcept;
[codeblock:end]

[enter:description]
[para:remarks]
[:en]
Called by the implementation when exception handling must be abandoned for any
of several reasons ([#except.terminate]), in effect immediately after throwing
the exception.

[:en]
May also be called directly by the program.

[para:effects]
[:en]
Calls the current [`terminate_handler] function.

[enter:note]
[:en]
A default [`terminate_handler] is always considered a callable handler in this
context.
[exit:note]
[exit:description]
